_data/Phil9987/phil9987.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Philip Junker
+institution: ETH Zurich
+profile_pic: IMG_20200525_170026_078.jpg
+quote: The block of granite which was an obstacle in the path of the weak, becomes a stepping stone in the path of the strong. 
```

Comment:
Quote must **not** be longer than 100 characters, yours is **119** characters.

Reply:
Shortened in latest commit.